pyproject.toml

```diff
@@ -96,7 +96,8 @@ test-min = [
     "joblib",
     "boltons",
     "scanpy>=1.10",
-    "httpx",                # For data downloading
+    # TODO: Is 1.0dev1 a real pre-release? https://pypi.org/project/httpx/#history
```

Comment:
I’m pretty sure that as far as Python package installers are concerned, “pre-release” means “Is on PyPI and doesn’t have a stable version”, so yes.

Do you mean that “dev1” isn’t a version you’d make a release of? Well, they apparently do!

Reply:
I commented here: https://github.com/encode/httpx/issues/3602